packages/react-components/react-motion/stories/src/Tokens/Cards.tsx

```diff
@@ -0,0 +1,266 @@
+  point: {
+    gridArea: 'point',
+
+    width: 'var(--point-size)',
+    height: 'var(--point-size)',
+
+    backgroundColor: tokens.colorBrandBackground,
+    clipPath: 'polygon(25% 0%, 100% 0%, 100% 100%, 25% 100%, 0% 50%)',
+
+    animationName: {
+      '0%': { transform: 'translateY(calc(var(--container-size) - var(--point-size)))' },
+      '60%, 100%': { transform: 'translateY(0)' },
+    },
+    animationDuration: '2s',
+    animationIterationCount: 'infinite',
+    animationFillMode: 'forwards',
+  },
+
+  graph: {
+    gridArea: 'graph',
+    position: 'relative',
+  },
+  graphX: {
+    color: tokens.colorNeutralStroke1,
+    fontFamily: tokens.fontFamilyMonospace,
+    position: 'absolute',
+    left: '10px',
+    top: '10px',
+  },
+  graphT: {
+    color: tokens.colorNeutralStroke1,
+    fontFamily: tokens.fontFamilyMonospace,
+    position: 'absolute',
+    bottom: '10px',
+    right: '10px',
+  },
+
+  duration: {
+    placeSelf: 'center',
+
+    width: '128px',
+    height: '4px',
+    margin: '64px auto',
+    backgroundColor: tokens.colorNeutralForeground3,
+
+    animationName: {
+      from: { transform: 'rotate(0)' },
+      to: { transform: 'rotate(180deg)' },
+    },
+    animationIterationCount: 'infinite',
+  },
+  svg: {
+    borderLeft: `1px solid ${tokens.colorNeutralStroke1}`,
+    borderBottom: `1px solid ${tokens.colorNeutralStroke1}`,
+
+    height: 'var(--container-size)',
+    width: 'var(--container-size)',
+  },
+
+  title: {
+    gridArea: 'title',
+    justifySelf: 'center',
+    margin: '0 20px',
+
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '5px',
+
+    fontFamily: tokens.fontFamilyMonospace,
+    fontSize: tokens.fontSizeBase400,
+    padding: '5px 10px',
+    backgroundColor: tokens.colorNeutralBackground3,
+    borderRadius: tokens.borderRadiusMedium,
+  },
+  name: {
+    textAlign: 'center',
+  },
+  value: {
+    fontSize: tokens.fontSizeBase200,
+    fontStyle: 'italic',
+    whiteSpace: 'nowrap',
+    textAlign: 'center',
+  },
+});
+
+function getPointsFromEasing(value: string) {
+  if (value.startsWith('cubic-bezier')) {
+    const points = value.slice(13, -1).split(',');
+
+    if (points.length !== 4) {
+      throw new Error('Invalid cubic-bezier() value');
+    }
+
+    return points.map(v => parseFloat(v) * 100) as [number, number, number, number];
+  }
+
+  throw new Error('Only cubic-bezier() easing is supported');
+}
+
+const MotionCurveCard: React.FC<{ animationEnabled: boolean; tokenName: string; tokenValue: string }> = props => {
+  const { animationEnabled, tokenName, tokenValue } = props;
+
+  const classes = useCardClasses();
+  const easingPoints = getPointsFromEasing(tokenValue);
+
+  return (
+    <div className={classes.container}>
+      <div className={classes.graph}>
+        <svg className={classes.svg} viewBox="0 0 100 100">
+          <path
+            d={`M 0 100 C ${easingPoints[0]} ${easingPoints[1]}, ${easingPoints[2]} ${easingPoints[3]}, 100 0`}
```

Comment:
The graphs are currently wrong:
<img width="266" alt="image" src="https://github.com/user-attachments/assets/a55b301b-b389-4890-b71d-f5b9a590d9a4">

You need to invert y axis:
```suggestion
            d={`M 0 100 C ${easingPoints[0]} ${100 - easingPoints[1]}, ${easingPoints[2]} ${100 - easingPoints[3]}, 100 0`}
```
<img width="264" alt="image" src="https://github.com/user-attachments/assets/35f6ac34-bb74-4912-8f00-3093c383eab6">